recipes/xyzspaces/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xyzspaces" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/heremaps/xyz-spaces-python/archive/{{ version }}.tar.gz
+  sha256: 060f023a9711d0d6abe1381ec2723ee8ec0c6551f24a413098295bf8b58121ed
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

Comment:
conda-forge isn't building for pytho<3.6, so you can remove the skip and use `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

you can always add the version restrictions under `requirements` (i.e change `python` to `python  >=3.6` under `host` and `run`)